x-pack/test/api_integration/apis/management/ingest_pipelines/lib/fixtures.ts

```diff
@@ -21,7 +21,7 @@ interface Pipeline {
   metadata?: Metadata;
 }
 
-interface IngestPutPipelineInternalRequest extends Omit<IngestPutPipelineRequest, 'id'> {
+export interface IngestPutPipelineInternalRequest extends Omit<IngestPutPipelineRequest, 'id'> {
```

Comment:
I can't see that this is used outside of this file?

Reply:
same as on the other case, it was a fix for a typescript error related to unaccessible types - I'm now verifying if these are still needed (it's just very very time consuming :( )

Reply:
This is the category of errors that shows up when some of these types are not exported:
```
 proc [tsc] x-pack/test/api_integration/services/ingest_pipelines.ts:14:17 - error TS4058: Return type of exported function has or is using name 'IngestPutPipelineInternalRequest' from external module "/Users/alex/Git/kibana/x-pack/test/api_integration/apis/management/ingest_pipelines/lib/fixtures" but cannot be named.
```